tools/checked-c-convert/utils/run.py

```diff
@@ -56,8 +56,8 @@ def runMain(args):
   args.append(prog_name)
   args.extend(DEFAULT_ARGS)
   args.extend(list(s))
-  f = open('bla', 'w')
-  f.write(" ".join(args))
+  f = open('convert.sh', 'w')
+  f.write(" \\\n".join(args))
```

Comment:
What about `os.linesep`?

Reply:
Neat, a new Python thing that I didn't know about! https://docs.python.org/2/library/os.html#os.linesep says not to use it when using this to write to text files, though. So I assume it's only supposed to be for files opened with 'wb' ?

Reply:
On further reading, I suspect this is something that I have just been cargo culting without fully understanding how it works. You're right. I was just worried that on Windows, if this ever switched to outputting `.bat` files, the newline representation would have to be changed, but the documentation I read that you linked seems to indicate that is not the case.

Reply:
Yes I thought the same about Windows until I read the docs.